pkg/kubectl/resource_printer.go

```diff
@@ -244,20 +246,20 @@ func (h *HumanReadablePrinter) HandledResources() []string {
 	return keys
 }
 
-var podColumns = []string{"POD", "IP", "CONTAINER(S)", "IMAGE(S)", "HOST", "LABELS", "STATUS", "CREATED", "MESSAGE"}
-var podTemplateColumns = []string{"TEMPLATE", "CONTAINER(S)", "IMAGE(S)", "PODLABELS"}
-var replicationControllerColumns = []string{"CONTROLLER", "CONTAINER(S)", "IMAGE(S)", "SELECTOR", "REPLICAS"}
-var serviceColumns = []string{"NAME", "LABELS", "SELECTOR", "IP(S)", "PORT(S)"}
-var endpointColumns = []string{"NAME", "ENDPOINTS"}
+var podColumns = []string{"NAMESPACE", "POD", "IP", "CONTAINER(S)", "IMAGE(S)", "HOST", "LABELS", "STATUS", "CREATED", "MESSAGE"}
```

Comment:
That's actually not a syntax we officially use, and conflicts with resourceType/name.  Can we punt on introducing the new format until we're ready to use it on the cli (after -q)?

> On May 13, 2015, at 10:05 AM, Brendan Burns <notifications@github.com> wrote:
> 
> In pkg/kubectl/resource_printer.go:
> 
> > @@ -244,20 +246,20 @@ func (h *HumanReadablePrinter) HandledResources() []string {
> >  	return keys
> >  }
> >  
> > -var podColumns = []string{"POD", "IP", "CONTAINER(S)", "IMAGE(S)", "HOST", "LABELS", "STATUS", "CREATED", "MESSAGE"}
> > -var podTemplateColumns = []string{"TEMPLATE", "CONTAINER(S)", "IMAGE(S)", "PODLABELS"}
> > -var replicationControllerColumns = []string{"CONTROLLER", "CONTAINER(S)", "IMAGE(S)", "SELECTOR", "REPLICAS"}
> > -var serviceColumns = []string{"NAME", "LABELS", "SELECTOR", "IP(S)", "PORT(S)"}
> > -var endpointColumns = []string{"NAME", "ENDPOINTS"}
> > +var podColumns = []string{"NAMESPACE", "POD", "IP", "CONTAINER(S)", "IMAGE(S)", "HOST", "LABELS", "STATUS", "CREATED", "MESSAGE"}
> Rather than adding an extra column (we're already pretty wide) can we just use the:
> 
> '<namespace>/<resource>' 
> syntax?
> 
> —
> Reply to this email directly or view it on GitHub.
>